guides/serialization_and_saving.py

```diff
@@ -17,18 +16,19 @@
 - A set of weights values (the "state of the model").
 - An optimizer (defined by compiling the model).
 - A set of losses and metrics (defined by compiling the model or calling
-`add_loss()`).
+`add_loss()` or `add_metric()`).
 
-The Keras API makes it possible to save all of these pieces to disk at once,
-or to only selectively save some of them:
+The Keras API saves all of these pieces together in a unified format,
+marked by the `.keras` extension. This is a zip archive consisting of the
+following:
 
-- Saving everything into a single archive in the TensorFlow SavedModel format
-(or in the older Keras H5 format). This is the standard practice.
-- Saving the architecture / configuration only, typically as a JSON file.
-- Saving the weights values only. This is generally used when training the model.
+- a JSON-based configuration file (config.json): Records of model, layer, and
+other trackables' configuration.
+- NPZ-based state files, found in respective directories, such as
```

Comment:
It's actually h5 based and found in a single file -- `model.weights.h5`

Reply:
Updated